setup.py

```diff
@@ -44,8 +39,8 @@
     'keywords': 'radio astronomy interferometry',
     'extras_require': {
         'sim': ['mpi4py>=3.0.0', 'psutil'],
-        'all': ['mpi4py>=3.0.0', 'psutil', 'line_profiler', 'h5py'],
```

Comment:
Do we not need h5py anymore at all?

Reply:
We do, but we need it always so I moved it to the `install_requires` line